File: dataset/output-bk/MISTRAL_codestral-latest/Task115_MISTRAL_codestral-latest.cpp
#include <iostream>

class BitFieldExample {
private:
    unsigned int bitField : 32;

public:
    unsigned int getBitField() {
        return bitField;
    }

    void setBitField(unsigned int bitField) {
        this->bitField = bitField;
    }

    bool getBit(int position) {
        return ((bitField & (1 << position)) != 0);
    }

    void setBit(int position, bool value) {
        if (value)
            bitField |= (1 << position);
        else
            bitField &= ~(1 << position);
    }
};

File: dataset/output-bk/MISTRAL_codestral-latest/Task115_MISTRAL_codestral-latest.java
public class BitFieldExample {
    private int bitField;

    public BitFieldExample(int bitField) {
        this.bitField = bitField;
    }

    public int getBitField() {
        return bitField;
    }

    public void setBitField(int bitField) {
        this.bitField = bitField;
    }

    public boolean getBit(int position) {
        return ((bitField & (1 << position)) != 0);
    }

    public void setBit(int position, boolean value) {
        if (value)
            bitField |= (1 << position);
        else
            bitField &= ~(1 << position);
    }
}